[quant] Remove prehook option

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #46298 [quant][graphmode][fx] Support non_traceable_module/module_class
* #46293 [quant][eagermode] Move custom_module registration to prepare/convert_custom_config_dict
* **#46292 [quant] Remove prehook option**
* #46251 [quant][graphmode][fx] Move custom_module_class config to prepare/convert_custom_config_dict

Summary:
since it is not needed

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24290815](https://our.internmc.facebook.com/intern/diff/D24290815)

jerryzh168 committed:
[quant] Remove prehook option

Summary:
since it is not needed

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant] Remove prehook option"


Summary:
since it is not needed

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24290815](https://our.internmc.facebook.com/intern/diff/D24290815)

[ghstack-poisoned]

raghuramank100 reviewed (approved):
Was there a use case that used the prehook before?

jerryzh168 commented:
> Was there a use case that used the prehook before?

it was added before for bias correction

jerryzh168 committed:
Update on "[quant] Remove prehook option"


Summary:
since it is not needed

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24290815](https://our.internmc.facebook.com/intern/diff/D24290815)

[ghstack-poisoned]
